lib/sycamore/sycamore/config.py

```diff
@@ -0,0 +1,21 @@
+from typing import Dict, Optional, Any
+
+
+class Config:
+
+    OPENSEARCH_CLIENT_CONFIG = "opensearch.client_config"
+    OPENSEARCH_INDEX_NAME = "opensearch.index_name"
+    OPENSEARCH_INDEX_SETTINGS = "opensearch.index_settings"
```

Comment:
maybe these constants belong in a more opensearchy namespace?

Reply:
can reorganize as they grow, for now figured having configs in 1 place is easier to manage